### ISSUE TITLE
select * from non-existent measurement can panic database

### ISSUE THREAD

**Issue by biskit** [influxdb/influxdb#3112]
just use a database and give name of unknown table. crashes server. basic test!

```
[abc influxdb]$ ./influx
Connected to http://localhost:8086 version FIXME
InfluxDB shell 0.9.0
> show databases
name: databases
---------------
name
collectd

> use collectd
Using database collectd
>
> select * from /.*/
> select * from cpu
ERR: Get http://localhost:8086/query?db=collectd&q=select+%2A+from+cpu: EOF
> select * from /.*/
ERR: Get http://localhost:8086/query?db=collectd&q=select+%2A+from+%2F.%2A%2F: dial tcp 127.0.0.1:8086: connection refused
> quit
ERR: Get http://localhost:8086/query?db=collectd&q=quit: dial tcp 127.0.0.1:8086: connection refused
```

**Comment by beckettsean** [comment]
@biskit Yes, as it turns out, if you ask the database to do something nonsensical, such as return the results from a measurement than doesn't exist, it fails. In this case it should fail more gracefully, but there is a `0` in front of the version number, meaning this is beta software. Considering what you paid for the database I think your snark is rude and is most certainly unappreciated.

**Comment by beckettsean** [comment]
I cannot reproduce this on my machine. I'm using 0.9.0 installed via Homebrew on OS X (Yosemite)

```
> show databases
name: databases
---------------
name
mydb
mydb1
mydb3

> use mydb3
Using database mydb3
> select * from does_not_exist
ERR: measurement not found: "mydb3"."default".does_not_exist
> 
```

No crash, the process is still up and running.

@biskit what OS are you running and how did you install InfluxDB?

**Comment by otoolep** [comment]
This has been fixed for 0.9.1.

**Comment by otoolep** [comment]
Tested against master.

```
{
    "results": [
        {
            "error": "measurement not found: \"foo\"..cpu"
        }
    ]
}
```

**Comment by biskit** [comment]
**Considering what you paid for the database I think your snark is rude and is most certainly unappreciated.**
I paid for it with my time.

**what OS are you running and how did you install InfluxDB?**
Centos 6.6 installed from rpm instructions on downloads page. latest available today
3b642659abedb934d95530b9890335cf  influxdb-latest-1.x86_64.rpm

**I believe this has been fixed for 0.9.1.**
Is download available and what's checksum on it

**Comment by beckettsean** [comment]
@biskit 0.9.1 is not available for download. It will be released in ~10 days.
